lib/reactive_table.js

```diff
@@ -492,6 +498,14 @@ Template.reactiveTable.helpers({
         if (this.showNavigation === 'always') return true;
         if (this.showNavigation === 'never') return false;
         return getPageCount.call(this) > 1;
+    },
+
+    'showRowCount': function() {
+        if(this.showRowCount)
```

Comment:
The helper and property are named the same so I suppose the template is using what is returned from the helper.